tests/unit/query_plan_create_set_remove_delete.cpp

```diff
@@ -1678,6 +1678,41 @@ TYPED_TEST(QueryPlanTest, SetPropertiesOnNull) {
   EXPECT_EQ(1, PullAll(*set_op, &context));
 }
 
+TYPED_TEST(QueryPlanTest, UpdateSetPropertiesFromMap) {
+  auto storage_dba = this->db->Access();
+  memgraph::query::DbAccessor dba(storage_dba.get());
+  // Add a single vertex. ( {property: 43})
+  auto vertex_accessor = dba.InsertVertex();
+  auto old_value = vertex_accessor.SetProperty(dba.NameToProperty("property"), memgraph::storage::PropertyValue{43});
+  EXPECT_EQ(old_value.HasError(), false);
+  EXPECT_EQ(*old_value, memgraph::storage::PropertyValue());
```

Comment:
Shouldn't this fail? Shouldn't old_value be equal to PropertyValue(43)

Reply:
No no, old value is empty, it didn't exist